vpp/vaapipostprocess_scaler.h

```diff
@@ -49,16 +49,21 @@ class VaapiPostProcessScaler : public VaapiPostProcessBase {
         BufObjectPtr filter; //send to va;
     };
 
-    struct ColorBalanceParam {
-        BufObjectPtr filter; //send to va;
-        int32_t level; //send to va
+    struct ColorBalanceProperty {
```

Comment:
this is useless changes, it will change many lines, but no benefits
Could we just keep the original name? and add m_colorBalanceFilter as a member of VaapiPostProcessScaler

Reply:
thanks @xuguangxin , But if we pick filter out from ColorBalanceParams, that will harm the structure style, Because  the similar structure DeinterlaceParams and ProcParams all contain filter, and there is no single filter member varible in this class too.